Sources/TuistDependencies/DependenciesGraph/DependenciesGraphController.swift

```diff
@@ -21,7 +21,7 @@ enum DependenciesGraphControllerError: FatalError, Equatable {
     var description: String {
         switch self {
         case .failedToDecodeDependenciesGraph:
-            return "Couldn't decode the DependenciesGraph from the serialized JSON file. Running `tuist dependencies fetch` should solve the problem."
+            return "Couldn't decode the DependenciesGraph from the serialized JSON file. Running `tuist fetch dependencies` should solve the problem."
```

Comment:
The command is `tuist dependencies fetch`. Why did you change it? 🤔

Reply:
It is actually changed, because now we fetch also the plugin tasks, so there is a `tuist fetch` command which does both

Reply:
oh I haven't known, thx for your comment :)